packages/web-components/fast-element/src/templating/repeat.ts

```diff
@@ -149,7 +149,7 @@ export class RepeatBehavior<TSource = any> implements Behavior, Subscriber {
         } else if (args[0].reset) {
             this.refreshAllViews();
         } else {
-            this.updateViews(args as Splice[]);
+            this.updateViews(args[0] as Splice);
```

Comment:
This is the one thing I'm not sure about. It seems that if there are multiple changes, that the `project` code can't necessarily merge those down to a single splice. There still might be multiple splices. We might need some tests that mix up a wider variety of array mutations to see this issue.

Reply:
Hey, good catch, after adding some combinations of array modification tests, this indeed needed to be reverted back to handle array of Splices.